src/VSCodeExtension/src/extension.ts

```diff
@@ -96,6 +187,85 @@ export async function activate(context: vscode.ExtensionContext) {
         }
     );
 
+    registerCommand(
+        context,
+        "quantum.submitJob",
+        async() => {
+            sendTelemetryEvent(EventNames.jobSubmissionStarted, {},{});
+            await getCredential(context);
+            requireDotNetSdk(dotNetSdkVersion).then(
+                dotNetSdk => submitJob(context, dotNetSdk, localSubmissionsProvider, credential,workSpaceStatusBarItem )
+            );
+        }
+    );
+
+    // registerCommand(
```

Comment:
Remove commented code. Did we decide not to have the possibility of disconnection? I think it might be confusing to keep a disconnection command if we can't log the user out of the Az CLI authentication anyway.